Brain/Brain.ino

```diff
@@ -82,8 +88,12 @@ void loop() {
       i++;
     }
     int action = decode_received_packet(String(command), &valve);
-    take_action(&valve, action);
-    make_packet(valve.id); // this might need to be valve->id
+    if (action != -1) {
+      take_action(&valve, action);
+      String packet = make_packet(valve.id); // this might need to be valve->id
```

Comment:
Just a note valve.id should be fine because id was just an int, not a pointer to a function.

Reply:
yup. thanks